example/spark/benchmark.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
Suggest file name: bootstrap-benchmarks.sh assuming we call the commands for user to add customize actions as bootstrap_commands.

Reply:
Have updated.